src/test/ui/attributes/key-value-non-ascii.stderr

```diff
@@ -2,10 +2,12 @@ error: non-ASCII character in byte constant
   --> $DIR/key-value-non-ascii.rs:3:19
    |
 LL | #[rustc_dummy = b"ﬃ.rs"]
-   |                   ^
-   |                   |
-   |                   byte constant must be ASCII
-   |                   help: use a \xHH escape for a non-ASCII byte: `\xFB03`
+   |                   ^ byte constant must be ASCII
+   |
+help: if you meant to use the UTF-8 encoding of 'ﬃ', use \xHH escapes
+   |
+LL | #[rustc_dummy = b"/xEF/xAC/x83.rs"]
+   |                   ^^^^^^^^^^^^
```

Comment:
I don't know why it does that; I am definitely using backslashes in the suggestion... This is not a regression, though, because the suggestion was also wrong before.

Reply:
It looks like it's happening during diagnostic emission, does this still happen if you run the test with `--error-format human-annotate-rs`?

Reply:
No, with this option, the entire suggestion disappears...

Reply:
Ah, maybe that option isn’t implemented fully yet.